go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/pkg/errors v0.8.0
 	github.com/shoenig/atomicfs v0.1.1
 	github.com/shoenig/httplus v0.0.0
+	github.com/shoenig/mockery3 v0.0.0-20180929002219-8b2d2c71cf53 // indirect
```

Comment:
we shouldn't need this in the go.mod file (it isn't imported directly by anything)
running `go mod tidy` automatically removes it:
```
[p1 mp (patch/gh-118)] $ go mod tidy
go: finding github.com/shoenig/mockery3 v0.0.0-20180929002219-8b2d2c71cf53
go: finding github.com/stretchr/testify v1.2.0
[p1 mp (patch/gh-118)] $ git diff
diff --git a/go.mod b/go.mod
index cc53341..b7af489 100644
--- a/go.mod
+++ b/go.mod
@@ -12,7 +12,6 @@ require (
        github.com/pkg/errors v0.8.0
        github.com/shoenig/atomicfs v0.1.1
        github.com/shoenig/httplus v0.0.0
-       github.com/shoenig/mockery3 v0.0.0-20180929002219-8b2d2c71cf53 // indirect
```